lang/java/avro/src/test/java/org/apache/avro/TestSchemaValidation.java

```diff
@@ -77,6 +77,8 @@
       new ReaderWriter(BYTES_UNION_SCHEMA, STRING_UNION_SCHEMA),
       new ReaderWriter(DOUBLE_UNION_SCHEMA, INT_FLOAT_UNION_SCHEMA),
 
+      new ReaderWriter(NULL_INT_ARRAY_UNION_SCHEMA, INT_ARRAY_SCHEMA),
+
```

Comment:
We should add a test for union {null, map} since this fixes dealing with that case as well.

Reply:
Done.